src/ecs/commands/runCommandInContainer.ts

```diff
@@ -30,35 +30,19 @@ export async function runCommandInContainer(
         return
     }
 
-    // Check to see if there are any deployments in process
-    const deployments = (await node.ecs.describeServices(node.clusterArn, [node.serviceName]))[0].deployments
-    if (deployments) {
-        for (let i = 0; i < deployments.length; i++) {
-            if (deployments[i].status === 'ACTIVE' || deployments[i].rolloutState === 'IN_PROGRESS') {
-                window.showWarningMessage(
-                    localize(
-                        'AWS.command.ecs.runCommandInContainer.warnDeploymentInProgress',
-                        'A deployment for this service may still be in progress. Not all containers may be running.'
-                    )
-                )
-            }
-            break
-        }
+    // Check to see if there are any deployments in progress
+    if(await isDeploymentInProgress(node)) {
+        window.showWarningMessage(
+            localize(
+                'AWS.command.ecs.runCommandInContainer.warnDeploymentInProgress',
+                'A deployment for this service may still be in progress. Not all containers may be running.'
```

Comment:
this message is too vague.

```suggestion
                'This service is currently deploying.'
```